.run/Run Plugin.run.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
We'll have to fix the gitignore situation sooner or later so you don't blanket-exclude `.idea` :D